### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by rpire**
## Changes made:
-  Added the `email` column to the `owners` table.
- Inserted required values to the `visits` and `owners` tables.
- Ran `EXPLAIN ANALYZE` with the required queries and took record of execution times with screenshots.
- Added screenshot files from before the optimization.

### Screenshots

> First query screenshot before optimization.
BQUERY#1

> First query screenshot after optimization.
AQUERY#1

> Second query screenshot before optimization.
BQUERY#2

> Second query screenshot after optimization.
AQUERY#2

> Third query screenshot before optimization.
BQUERY#3

> Third query screenshot after optimization.
AQUERY#3

### CODE REVIEW

**Comment by DammyShittu** [comment]
Dear Reviewer, 
We had an issue while pushing code changes. We mistakenly pushed to the `dev` branch and had to delete it and recreate it, hence the high number of commits.
Thanks.

File: schema.sql [hunk 1]
```diff
@@ -9,3 +9,83 @@ CREATE TABLE animals (
...
+
+-- Performance Audit
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX animal_id_desc ON visits (animal_id DESC);
+
+CREATE INDEX vet_id_desc ON visits (vet_id DESC);
+
+CREATE INDEX email_desc ON owners (email DESC);
```
Comment (DeliceLydia) [inline schema.sql]: ## Optional
As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:
`CREATE INDEX ON owners ((lower(email)));`

**Review by DeliceLydia — APPROVED** [review]
# Congratulations! 🎉🎉

## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rpire** [comment]
Hello, @DeliceLydia 👋

Thank you for reviewing our work. We will make sure to implement your suggestions before merging these changes into the `dev` branch. 🚀

Cheers and Happy coding to you too! 🤓

**Comment by DammyShittu** [comment]
Dear @DeliceLydia,

Thank you for the review. As suggested, we have implemented the optional changes.

Cheers 🚀

**Pull request merged**